test_snappy.py

```diff
@@ -29,7 +29,7 @@
 import os
 import sys
 import random
-import snappy
+from snappy import snappy
```

Comment:
Will all users of the new structure need to do this? 
Could not the `__init__.py` import everything one might need from the module, so that we don't force a code change on users of snappy?

Reply:
@martindurant short answer is no, the public API is unchanged.

In this case, the tests expect access to internal `snappy` variables (`_CHUNK_MAX` for example) which didn't make sense to include in the API and import in `__init__.py`. My first approach was to import the internal `snappy` module so the tests could access those internal variables.

I've updated the PR to import `snappy` at the top level and access the hidden variables via `snappy.snappy`. Let me know what you think.

Reply:
That's fine with me, so long as the PR is transparent for users.

Reply:
Agreed; the current API is preserved.